src/selectors/visibleColumnBreakpoints.js

```diff
@@ -101,10 +102,20 @@ export function getColumnBreakpoints(pausePoints, breakpoints, viewport) {
     ({ location: { line } }) => lineCount[line] > 1
   );
 
-  return columnBreakpoints.map(({ location }) => ({
-    location,
-    enabled: isEnabled(location, breakpointMap)
-  }));
+  return columnBreakpoints.map(({ location }) => {
+    const foundBreakpoint = breakpoints.find(breakpoint => {
+      return (
+        breakpoint.location.line === location.line &&
+        breakpoint.location.column === location.column
+      );
```

Comment:
lets use lodash's isEqual

Reply:
We're comparing these objects so that seems like overkill:

<img width="1109" alt="nahh" src="https://user-images.githubusercontent.com/46655/49317882-e69e5d00-f4bb-11e8-85a8-b0ec39691828.png">